client_taxi_driver/androidApp/src/androidMain/assets/bing_map/map/script.js

```diff
@@ -0,0 +1,93 @@
+var map,directionsManager;
+function getMap() {
+map = new Microsoft.Maps.Map('#myMap', {
+    credentials: 'Access_token',
+    center: new Microsoft.Maps.Location(0,0),
```

Comment:
why this line?
i removed it from delivery map as it casues a new point on the center of the map